src/plugins/plot/stackedPlot/mixins/objectStyles-mixin.js

```diff
@@ -132,12 +132,6 @@ export default {
                     }
                 }
             });
-
```

Comment:
I'm a little confused by this. Any idea what it did, and why it's in `../stackedPlot`? Appears to be the same logic as in `objectView` but I don't understand why.

Reply:
Yeah I also not sure, adding @shefalijoshi (can you also review this? TIA)

Reply:
@nikhilmandlik Stacked plots were not showing conditional styles. For them to work, we had to work on each item inside of the stacked plot as they are not ObjectViews.

Reply:
hmm, I think current condition on stack plot is also showing correctly,
It only applies to first item.
<img width="691" alt="Screen Shot 2022-05-02 at 2 19 35 PM" src="https://user-images.githubusercontent.com/1292612/166329558-3a3521a7-3559-4599-b84a-7a2965c11f78.png">

Reply:
But why the specific `conditionalLabel` logic? Does that apply to stacked plots at all, or was that just because it's a mixin being used in more than one place?